modules/porous_flow/test/tests/dirackernels/frompps.i

```diff
@@ -0,0 +1,146 @@
+# Test PorousFlowPointSourceFromPostprocessor DiracKernel
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 2
+  bias_x = 1.1
+  bias_y = 1.1
+  ymax = 1
+  xmax = 1
+[]
+
+[GlobalParams]
+  PorousFlowDictator = dictator
+[]
+
+[Functions]
+  [./mass_flux_fn]
+    type = PiecewiseConstant
+    direction = left
+    xy_data = '
+      0    0
+      100  -0.1
+      300  0
+      600  -0.1
+      1400 0
+      1500 0.2
+      2000 0.2'
+  [../]
+[]
+
+[Variables]
+  [./pp]
+  [../]
+[]
+
+[Kernels]
+  [./mass0]
+    type = PorousFlowMassTimeDerivative
+    fluid_component = 0
+    variable = pp
+  [../]
+[]
+
+[UserObjects]
+  [./dictator]
+    type = PorousFlowDictator
+    porous_flow_vars = pp
+    number_fluid_phases = 1
+    number_fluid_components = 1
+  [../]
+  [./pc]
+    type = PorousFlowCapillaryPressureVG
+    m = 0.5
+    alpha = 1e-7
+  [../]
```

Comment:
If you can use `PorousFlow1PhaseFullySaturated` then you don't need this.  If I'm mistaken and you can't use `PorousFlow1PhaseFullySaturated` then you can use
```
[./pc]
  type = PorousFlowCapillaryPressureConst
[../]
```
instead of what you have here